docs/docs/articles/run-tests-with-github-actions.md

```diff
@@ -1,136 +1,114 @@
 # Run Tests with GitHub Actions
```

Comment:
I don't think this file should be replaced at all, maybe there could be a note, like "If you need more advanced management, use **Testkube Action** instead"

Reply:
oh I got confused with all the actions. I will put it back. Thanks!